bin/create_decks

```diff
@@ -25,17 +25,17 @@ function copy_images () {
 
 function create_index() {
     {
-        echo "<h1> All decks from farbenmeer</h1>"
+        echo "# All decks from farbenmeer"
         for deck in $DECKS; do
         if [[ "$deck" == "sample" ]]; then
             continue
         fi
-        echo '<a href="'"$deck"'">'"$deck"'</a></br>'
+        printf '* ['$deck']('$deck')\n'
```

Comment:
Are you sure this is working? afaik: single quotes will not expand variables (`$deck`)

Reply:
Yup, it is working. But to be consistent I will change it.